lib/dor-services.rb

```diff
@@ -31,29 +29,15 @@ def load_instance(pid)
     # @param [String] pid The object's PID
     # @return [Object] the ActiveFedora-modeled object
     def find(pid, opts = {})
-      opts[:rows] = 1 # we know we are going to just pay attention to the .first doc anyway
-      find_all(%(id:"#{pid}"), opts).first || load_instance(pid)
+      load_instance(pid)
```

Comment:
Yes, it's one of them... and not necessarily a bad idea, just seems unnecessary if we immediately turn around and make other requests.